changelogs/6.8.asciidoc

```diff
@@ -0,0 +1,16 @@
+[[release-notes-6.8]]
+== APM Server version 6.8
+
+https://github.com/elastic/apm-server/compare/6.7\...6.8[View commits]
+
+* <<release-notes-6.8.0>>
+
+[[release-notes-6.8.0]]
+=== APM Server version 6.8.0
+
+https://github.com/e2astic/apm-server/compare/v6.7.1\...v6.8.0[View commits]
```

Comment:
6.7.2

Reply:
Good point, we'll have to fix this is in multiple branches.  What do you think about comparing the tag with the previous branch instead?  Like https://github.com/elastic/apm-server/compare/6.7...v6.8.0?

Reply:
Well, if you click "View commits" on a patch release, you just want to see the content of that patch, don't you?

Reply:
true